tests/js/spec/views/organizationGroupDetails/__snapshots__/groupSimilar.spec.jsx.snap

```diff
@@ -1502,11 +1502,9 @@ exports[`Issues Similar View renders with mocked data 1`] = `
                                             avatar={true}
                                           >
                                             <AutoSelectText
-                                              avatar={true}
```

Comment:
o maybe not

Reply:
Yeah I was trying to remove the DOM warnings about a div getting an invalid prop now that AutoSelectText is in typescript.